src/conditional-flow/multiple-conditions.js

```diff
@@ -1,19 +1,17 @@
 // This function should return true if num is greater
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
-function isInRange (num, lower, upper) {
-
+function isInRange(num, lower, upper) {
+  return num >= lower && num <= upper ? true : false;
```

Comment:
The ternary operator here is not needed. `num >= lower && num <= upper` will resolve to either a `true` or `false` value. So this can be simplified to 
```js
return num >= lower && num <= upper
```